README.md

```diff
@@ -223,6 +225,14 @@ redcarpet:
     - with_toc_data
 ```
 
+### Font configuration
+
+The default fonts can be configured in the `_data/fonts.yml` file.
+
+### Color configuration
+
+The colors for components can be configured in the `_data/colors.yml` file. 
```

Comment:
If I'm reading correctly, it looks like `fonts.yml` and `colors.yml` have been replaced by `theme.yml`, correct?

Reply:
Yeah, that's right! 👍